src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkList.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openloadflow.network.impl;
+
+import com.powsybl.openloadflow.network.LfNetwork;
+
+import java.util.List;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public interface LfNetworkList extends AutoCloseable {
+
+    List<LfNetwork> getList();
```

Comment:
we're not so much interested in the list itself, but in the largest element. Shouldn't we add a `Optional<LfNetwork> getLargestNetwork()`? If not we should at least add in the javadoc that the list should be sorted.

Reply:
Ok, added